core/templates/dev/head/pages/exploration-editor-page/services/learner-answer-details-data.service.ts

```diff
@@ -36,12 +36,23 @@ angular.module('oppia').factory('LearnerAnswerDetailsDataService', [
       '<entity_id>');
 
     var _fetchLearnerAnswerInfoData = function() {
-      var learnerAnswerInfoDataUrl = UrlInterpolationService.interpolateUrl(
-        LEARNER_ANSWER_INFO_DATA_URL, {
-          entity_type: 'exploration',
-          entity_id: _expId});
-
-      return $http.get(learnerAnswerInfoDataUrl);
+      var userIsLoggedIn = false;
```

Comment:
We have `UserService` which can be helpful here*

Reply:
I got this code from the `UserService`. We do not have any function in `UserService` which return user info or info about user being logged in.

Reply:
Oh, I got this code from `UserService`. I missed the function which can be called instead. I will make the updates.